test/self/formats.py

```diff
@@ -73,3 +73,36 @@ def test_frame_reader_Y410(width, height):
     assert yf(y).all()
     assert uf(u).all()
     assert vf(v).all()
+
+@slash.parametrize(("width", "height"), itertools.product([175, 176, 177], [143, 144, 145]))
+def test_frame_reader_Y210(width, height):
+  testfile = get_media()._test_artifact(
+    "test_frame_reader_Y210_{}x{}.Y210".format(width, height))
+  #0xffc0 0xf0c0 0xdbc0 0x1dc0
+  oy0 = 65472; ou = 61632; oy1 = 56256; ov = 7616
+  dword1 = (ou << 16) | oy0
+  dword2 = (ov << 16) | oy1
+  b1 = dword1.to_bytes(4, "little")
+  b2 = dword2.to_bytes(4, "little")
+  width2  = (width + (width & 1)) >> 1;
+  size = width2 * height
+  with open(testfile, "wb") as fd:
+    for i in range(size):
+      fd.write(b1)
+      fd.write(b2)
```

Comment:
How about using size = width * height.  Then only write 1 dword per iteration of the loop, alternating between dword1 and dword2.